### PR TITLE
OpenShift template readme updates for pfioh

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
cc @danmcp Same a pman changes. 
cc @awalkaradi95moc

### CODE REVIEW

File: openshift/README.rst [hunk 1]
```diff
@@ -2,17 +2,15 @@
...
+We intend to use Swift Object store as default backend for pfioh in OpenShift template.
 
 ##############
 Swift Object Store
 ##############
 
 The OpenStack Object Store project, known as Swift, offers cloud storage software so that you can store and retrieve lots of data with a simple API. It's built for scale and optimized for durability, availability, and concurrency across the entire data set. Swift is ideal for storing unstructured data that can grow without bound. 
 
-To enable Swift Object store option for pfioh, start pfioh with --swift-storage option
+To enable Swift Object store option for pfioh, start pfioh with --swift-storage option.(This has been already taken care of if you are using OpenShift template available in this repo).
```
Comment (danmcp) [inline openshift/README.rst]: Remove the . after option

File: openshift/README.rst [hunk 2]
```diff
@@ -35,7 +33,7 @@ The credentials file for Swift should be stored in a **secret**, mounted at /etc
     osProjectName    = 
 
 **************
-Creating a secret
+Creating a secret and running pfioh.
```
Comment (danmcp) [inline openshift/README.rst]: Remove the . at the end.

File: openshift/README.rst [hunk 3]
```diff
@@ -2,17 +2,15 @@
 Setup:
 ##############
 
-.. code-block:: bash
-
-    oc new-app openshift/pfioh-openshift-template.json
+We intend to use Swift Object store as default backend for pfioh in OpenShift template.
```
Comment (danmcp) [inline openshift/README.rst]: in the OpenShift template.

**Comment by ravisantoshgudimetla** [comment]
Done @danmcp.

File: openshift/README.rst [hunk 4]
```diff
@@ -2,17 +2,15 @@
...
+We intend to use Swift Object store as default backend for pfioh in the OpenShift template.
 
 ##############
 Swift Object Store
 ##############
 
 The OpenStack Object Store project, known as Swift, offers cloud storage software so that you can store and retrieve lots of data with a simple API. It's built for scale and optimized for durability, availability, and concurrency across the entire data set. Swift is ideal for storing unstructured data that can grow without bound. 
 
-To enable Swift Object store option for pfioh, start pfioh with --swift-storage option
+To enable Swift Object store option for pfioh, start pfioh with --swift-storage option(This has been already taken care of if you are using OpenShift template available in this repo).
```
Comment (danmcp) [inline openshift/README.rst]: add a space after option

Comment (danmcp) [reply openshift/README.rst]: and start This with a lower case t

File: openshift/README.rst [hunk 5]
```diff
@@ -2,17 +2,15 @@
...
+We intend to use Swift Object store as default backend for pfioh in the OpenShift template.
 
 ##############
 Swift Object Store
 ##############
 
 The OpenStack Object Store project, known as Swift, offers cloud storage software so that you can store and retrieve lots of data with a simple API. It's built for scale and optimized for durability, availability, and concurrency across the entire data set. Swift is ideal for storing unstructured data that can grow without bound. 
 
-To enable Swift Object store option for pfioh, start pfioh with --swift-storage option
+To enable Swift Object store option for pfioh, start pfioh with --swift-storage option (this has been already taken care of if you are using OpenShift template available in this repo).
```
Comment (danmcp) [inline openshift/README.rst]: using the OpenShift template

**Pull request merged**